cicd/build.sh

```diff
@@ -23,6 +23,8 @@ CSM_PATH="${CORTX_PATH}csm"
 DEBUG="DEBUG"
 INFO="INFO"
 PROVISIONER_CONFIG_PATH="${CORTX_PATH}provisioner/generated_configs"
+CORTX_UNSUPPORTED_FEATURES_PATH="${CORTX_PATH}schema/unsupported_features.json"
+BRAND_UNSUPPORTED_FEATURES_PATH="/config/csm/unsupported_features.json"
```

Comment:
Looks like an absolute path. Is there a /config directory in our system?

Reply:
-l argument will have the initial path to the repo dir for seagate-ldr. This path will be appended to that initial path to get the path to unsupported_features.json in the seagate-ldr repo dir.

Reply:
yes. That's why it's confusing. This should be defined as following - 

BRAND_UNSUPPORTED_FEATURES_PATH=_"config/csm/unsupported_features.json"_

It tells the code reader that it is a relative path and use it by prepending "/" whereever it's required.

Reply:
Done.